webpack/src/ForemanDiscovery/DiscoveredHosts/Components/EmptyState/EmptyState.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import ForemanEmptyState from "foremanReact/components/common/EmptyState";
+
+const EmptyState = (props) => {
+  const description =
+    "No discovered hosts found in this context. This page shows discovered bare-metal or virtual nodes waiting to be provisioned.";
```

Comment:
should it be split into multi-lines? also.. can we pass a component to the `ForemanEmptyState`?
can you add translations?

Reply:
I think we should not split here, rather must do a split in the core PR. What do you think?